InterfaceParser/parsers/json_rpc.py

```diff
@@ -0,0 +1,98 @@
+"""JSON RPC parser.
+
+Contains parser for JSON RPC XML format.
+
+"""
+from parsers.parse_error import ParseError
```

Comment:
If this generator is only used by HMI_API.xml in core can we rename it to something less ambiguous than json_rpc.py?

Reply:
what would you suggest, "hmi_api.py"?

Reply:
thank you